projects/github-actions/repo-gardening/src/tasks/triage-issues/index.js

```diff
@@ -11,6 +14,112 @@ const sendSlackMessage = require( '../../utils/slack/send-slack-message' );
 
 /* global GitHub, WebhookPayloadIssue */
 
+/**
+ * Request a list of matching labels from Open AI that can be applied to the issue,
+ * based on the issue contents.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} title   - Issue title.
+ * @param {string} body    - Issue body.
+ *
+ * @return {Promise<Object>} Promise resolving to an object of labels to apply to the issue, and their explanations.
+ */
+async function fetchOpenAiLabelsSuggestions( octokit, owner, repo, title, body ) {
+	const suggestions = { labels: [], explanations: {} };
+
+	// Get all the Feature and Feature Group labels in the repo.
+	const pattern = /^(\[Feature\]|\[Feature Group\])/;
+	const repoLabels = await getAllLabels( octokit, owner, repo, pattern );
+
+	// If no labels are found, bail.
+	if ( repoLabels.length === 0 ) {
+		debug( 'triage-issues: No labels found in the repository. Aborting OpenAI request.' );
+		return suggestions;
+	}
+
+	const prompt = `Here is the issue title. It is the most important part of the text you must analyse:
+${ title }
+
+Here is the issue body:
+${ body }
+
+Here are the existing labels and their descriptions:
+${ repoLabels.join( ', ' ) }
```

Comment:
This doesn't seem like it's going to do what you intend. As I read it, it'll do like this:

> Here are the existing labels and their descriptions:
> [object Object], [object Object], [object Object], [object Object], [object Object]

Seems like you might want to do something more like this?
```suggestion
${ repoLabels.map( label => '- ' + label.name + ': ' + label.description ).join( '\n' ) }
```
or maybe
```suggestion
${ repoLabels.map( label => '- ' + label.name + ': ' + ( label.description || '(no description)' ) ).join( '\n' ) }
```

Reply:
Ah yes, I missed that! 🤦 . 1214685ea02db944fd541aaf4b429038e317bda8 should help.